packages/module/src/MessageBar/MessageBar.test.tsx

```diff
@@ -0,0 +1,145 @@
+    expect(sendButton).toBeTruthy();
+    await userEvent.click(sendButton);
+    expect(spy).toHaveBeenCalledTimes(1);
+  });
+  it('can send via enter key', async () => {
+    const spy = jest.fn();
+    render(<MessageBar onSendMessage={spy} />);
+    const input = screen.getByRole('textbox', { name: /Send a message.../i });
+    await userEvent.type(input, 'Hello world');
+    expect(input).toHaveValue('Hello world');
+    await userEvent.type(input, '[Enter]');
+    expect(spy).toHaveBeenCalledTimes(1);
+  });
+  it('can show attach menu', async () => {
+    render(
+      <MessageBar
+        onSendMessage={jest.fn}
+        attachMenuProps={{
+          isAttachMenuOpen: true,
+          setIsAttachMenuOpen: jest.fn(),
+          onAttachMenuToggleClick: jest.fn(),
+          onAttachMenuInputChange: jest.fn(),
+          attachMenuItems: ATTACH_MENU_ITEMS
+        }}
+      />
+    );
+    expect(screen.getByRole('textbox', { name: /Filter menu items/i })).toBeTruthy();
+    expect(screen.getByRole('menuitem', { name: /auth-operator/i })).toBeTruthy();
+    expect(screen.getByRole('menuitem', { name: /Alerts/i })).toBeTruthy();
+    expect(screen.getByRole('menuitem', { name: /Events/i })).toBeTruthy();
+    expect(screen.getByRole('menuitem', { name: /Logs/i })).toBeTruthy();
+    expect(screen.getByRole('menuitem', { name: /YAML - Status/i })).toBeTruthy();
+    expect(screen.getByRole('menuitem', { name: /YAML - All contents/i })).toBeTruthy();
+  });
+  it('can toggle attach menu', async () => {
+    const attachToggleClickSpy = jest.fn();
+    render(
+      <MessageBar
+        onSendMessage={jest.fn}
+        attachMenuProps={{
+          isAttachMenuOpen: false,
+          setIsAttachMenuOpen: jest.fn(),
+          onAttachMenuToggleClick: attachToggleClickSpy,
+          onAttachMenuInputChange: jest.fn(),
+          attachMenuItems: ATTACH_MENU_ITEMS
+        }}
+      />
+    );
+    expect(screen.queryByRole('textbox', { name: /Filter menu items/i })).toBeFalsy();
+    expect(screen.queryByRole('menuitem', { name: /auth-operator/i })).toBeFalsy();
+    expect(screen.queryByRole('menuitem', { name: /Alerts/i })).toBeFalsy();
+    expect(screen.queryByRole('menuitem', { name: /Events/i })).toBeFalsy();
+    expect(screen.queryByRole('menuitem', { name: /Logs/i })).toBeFalsy();
+    expect(screen.queryByRole('menuitem', { name: /YAML - Status/i })).toBeFalsy();
+    expect(screen.queryByRole('menuitem', { name: /YAML - All contents/i })).toBeFalsy();
+    const attachButton = screen.getByRole('button', { name: 'Attach button' });
+    await userEvent.click(attachButton);
+    expect(attachToggleClickSpy).toHaveBeenCalledTimes(1);
+  });
+  it('can show stop button', () => {
+    render(<MessageBar onSendMessage={jest.fn} hasStopButton handleStopButton={jest.fn} />);
+    expect(screen.getByRole('button', { name: 'Stop button' })).toBeTruthy();
+  });
+});
```

Comment:
Could we add tests for the following props?

- alwayShowSendButton
- hasAttachButton
- hasMicrophoneButton
- handleStopButton
- handleAttach
- isSendButtonDisabled
- buttonProps